docs/prepare-a-shared-rp-development-environment.md

```diff
@@ -300,9 +300,20 @@ each of the bash functions below.
    import_certs_secrets
    ```
 
+   An additional key/certificate (cluster-mdsd) is also required in the
+   $KEYVAULT_PREFIX-svc key vault.  This is the client certificate for cluster
+   log forwarding to Geneva.
+
+   ```bash
```

Comment:
We should move this command in the helper file `hack/devtools/deploy-shared-env.sh` and call a function from here.